mlir/utils/jenkins/Jenkinsfile

```diff
@@ -111,13 +111,20 @@ void postProcessPerfRes(String chip) {
         alwaysLinkToLastBuild: false,
         keepAll: true,
         reportDir: 'build',
-        reportFiles: "${chip}_MLIR_Performance_Changes.html,${chip}_MLIR_vs_MIOpen.html",
+        reportFiles: "${chip}_MLIR_Performance_Changes.html,${chip}_MLIR_vs_MIOpen.html","${chip}_MLIR_Performance_Changes_Gemm.html","${chip}_MLIR_vs_rocBLAS.html"
```

Comment:
Jenkins is complaining the syntax here. Maybe you can put the file in the same quotes.